proto/pdpb.proto

```diff
@@ -204,6 +206,19 @@ message GetRegionByIDRequest {
 
 // Use GetRegionResponse as the response of GetRegionByIDRequest.
 
+message ScanRegionsRequest {
+    RequestHeader header = 1;
+
+    bytes region_key = 2;
+    int32 limit = 3;
+}
+
+message ScanRegionsResponse {
+    ResponseHeader header = 1;
+    repeated metapb.Region regions = 2;
+    repeated metapb.Peer leaders = 3;
```

Comment:
How about region has no leader?

Reply:
It will add empty value to make regions and leaders are matched.